src/mockdata/mockTimeEntry.ts

```diff
@@ -0,0 +1,15 @@
+import { TimeEntry } from '../types/visualisation';
+import { addDays } from 'date-fns';
+
+/**
+ * Generate sample time entries for use in vis. editor.
+ *
+ * @param size Size of the returned array.
+ * @param min Minimum value of time entry value.
+ * @param max Maximum value of time entry value.
+ */
+export default function (size: number, [min, max]: [number, number] = [0, 100]): TimeEntry[] {
+    const values = new Array(size).fill(null).map(() => Math.random() * (max - min) + min);
+    const timeseries = Object.keys(new Array(size).fill(null)).map((i) => addDays(new Date(), Number(i)));
+    return values.map((e, i) => ({ value: e, time: timeseries[i].toISOString() }));
+}
```

Comment:
Instead of this, couldn't you use the the makeTimeseries function that already exists in mocks.ts? And just map that to TimeEntry with the mapping function?

Reply:
[This](https://github.com/IT2901-SINTEF01/frontend/blob/c9790fdf5f095baedb29415c57f42b8ce54fa363/src/mockdata/mocks.ts#L46) function creates a timeseries object for use with forecast data. The name `makeTimeseries` might be slightly misleading as it's meant for only one use case (or so it seems).